src/facade/contentFetcherFacade.ts

```diff
@@ -0,0 +1,47 @@
+import {JsonObject} from '@croct/json';
+import {formatCause} from '../error';
+import {ContentFetcher, FetchResponse, FetchOptions as BaseOptions} from '../contentFetcher';
+import {ContextFactory} from './evaluatorFacade';
+import {fetchOptionsSchema as optionsSchema} from '../schema';
+
+export type FetchOptions = Omit<BaseOptions, 'context'> & {
+    attributes?: JsonObject,
+};
+
+function validate(options: unknown): asserts options is FetchOptions {
+    if (typeof options !== 'object' || options === null) {
+        throw new Error('The options must be an object.');
+    }
```

Comment:
Won't this already be checked by the validate below?

Reply:
It was to make the message more friendly.